### PR TITLE
feat: embed proxy endpoints

### PR DESCRIPTION
**Pull request by olav**
https://linear.app/unleash/issue/1-154/embedded-proxy-add-apifrontend-endpoints

### CODE REVIEW

File: src/lib/proxy/create-context.test.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+test('will not blow up if properties is an array', () => {
+    const context = createContext({
+        userId: '123',
+        tenantId: 'some-tenant',
+        region: 'eu',
+        properties: ['some'],
+    });
+
+    // console.log(context);
```
Comment (FredrikOseberg) [inline src/lib/proxy/create-context.test.ts:58]: removable

File: src/lib/proxy/create-context.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        environment,
+        userId,
+        sessionId,
+        remoteAddress,
+        properties: Object.assign({}, rest, properties),
+    };
+
+    // Clean undefined properties on the context
+    const cleanContext = Object.keys(context)
+        .filter((k) => context[k])
```
Comment (FredrikOseberg) [inline src/lib/proxy/create-context.ts:30]: Could we use full parameter names?

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]

File: src/lib/proxy/create-context.ts (hunk 2 above)
Comment (olav) [reply src/lib/proxy/create-context.ts:30]: See above.

File: src/lib/proxy/create-context.test.ts (hunk 1 above)
Comment (FredrikOseberg) [reply src/lib/proxy/create-context.test.ts:58]: We should have that as a discussion point.

**Pull request merged**